samples/Riok.Mapperly.Sample/CarDto.cs

```diff
@@ -0,0 +1,40 @@
+namespace Riok.Mapperly.Sample;
+
+public class CarDto
+{
+    public string Name { get; set; } = string.Empty;
+
+    public int NumberOfSeats { get; set; }
+
+    public CarColorDto Color { get; set; }
+
+    public ProducerDto? Producer { get; set; }
+
+    public List<TireDto>? Tires { get; set; }
+}
+
+public enum CarColorDto
+{
+    Black,
```

Comment:
Maybe introduce an undefined value here and map this enum by name to showcase the enum mapping capabilities?

Reply:
Added different values to showcase this